lib/vmpooler/pool_manager.rb

```diff
@@ -1,12 +1,17 @@
 module Vmpooler
   class PoolManager
-    def initialize(config, logger, redis, graphite=nil)
+    def initialize(config, logger, redis, graphite=nil, statsd=nil)
       $config = config
 
       # Load logger library
       $logger = logger
 
-      unless graphite.nil?
+      # statsd and graphite are mutex in the context of vmpooler
+      unless statsd.nil?
```

Comment:
I've got more to say about this in another comment I'm working on, but `nil` is already falsey, so you can just use the status of something that *might* be `nil` instead of explicitly checking for it.

```diff
 - unless graphite.nil?
 + if graphite
```